jenkins/debian/debos/rootfs.yaml

```diff
@@ -6,6 +6,8 @@
 {{- $test_overlay := .test_overlay -}}
 {{- $extra_packages_remove := or .extra_packages_remove -}}
 {{- $extra_files_remove := or .extra_files_remove -}}
+{{- $crush_image_options := or .crush_image_options "" -}}
```

Comment:
Please keep changes to the common files such as `rootfs.yaml` in a separate commit from the one adding the new file system.

Reply:
okay will send a new PR request with changes only from `rootfs.yaml`

Reply:
Its a two line change related to `crush_image_options`, should it be sent with https://github.com/kernelci/kernelci-core/pull/221 PR instead this one? PR 221 provides better context to this change.